maskrcnn_benchmark/modeling/roi_heads/mask_head/loss.py

```diff
@@ -27,17 +27,15 @@ def project_masks_on_boxes(segmentation_masks, proposals, discretization_size):
     assert segmentation_masks.size == proposals.size, "{}, {}".format(
         segmentation_masks, proposals
     )
-    # TODO put the proposals on the CPU, as the representation for the
```

Comment:
Actually I would rather have this here, but instead of a TODO as a `FIXME`, because this is currently a bottleneck in the training code

Reply:
Agreed.
I was thinking how could this step be speed up a bit:
```
1) Iterate
2) Crop
3) Resize
4) Stack
```
But since the `Crop` step is carried out differently for each entry, I didn't see a straightforward way to do this.
If you think that a simple Python multiprocessing thread could solve this then I would be more than happy to implement it - I am just curious if it would collide with the torch backend.